Source/core/resources/exchangeRates.json

```diff
@@ -0,0 +1,173 @@
+{  
```

Comment:
Any thoughts on how to keep this updated? Currency exchange rates are very volatile.